com.github.lainsce.notejot.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id": "com.github.lainsce.notejot",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.lainsce.notejot",
+    "rename-icon": "com.github.lainsce.notejot",
+    "rename-appdata-file": "com.github.lainsce.notejot.appdata.xml",
+    "build-options": {
+        "env": {
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=all",
```

Comment:
what do you need device=all for?

Reply:
I actually saw this in other elementary apps' flatpak json files, might be something that's needed. I couldn't complete the flatpak build and make it work without it. 🤔

Reply:
What errors do you get? notes-up, for example doesn't have it. https://github.com/flathub/com.github.philip_scott.notes-up/blob/master/com.github.philip_scott.notes-up.json

we needed it in spice-up to enable access to remote presentation devices, it's mostly if you require specific hardware support that doesn't yet have a portal.

Reply:
I guess it was a one-time error, I could build it now when I removed it. I'm going to do a new push without the line.